sdk/servicebus/azure-servicebus/azure/servicebus/_pyamqp/aio/_client_async.py

```diff
@@ -38,7 +38,6 @@
     AMQPException,
     MessageException
 )
-from ..constants import LinkState
```

Comment:
just confirming, this wasnt being used anywhere?

Reply:
yes, it's not being used anywhere in this file